mlir/test/python/execution_engine.py

```diff
@@ -807,6 +807,7 @@ def testDumpToObjectFile():
                 # because RTDyldObjectLinkingLayer::emit will try to resolve symbols before dumping
                 # (see the jitLinkForORC call at the bottom there).
                 shared_libs=[MLIR_C_RUNNER_UTILS],
+                enable_pic=True,
```

Comment:
I guess the intent of this "test" is to show that nothing breaks because there's no straightforward way to show the object file is indeed PIC (short of running a linker to make a shlib)?